guides/source/active_record_migrations.md

```diff
@@ -357,9 +374,9 @@ get to work!
 
 ### Creating a Table
```

Comment:
OK this threw me for a moment - specifically the difference between Generating Migrations and Writing Migrations. All of the below goes into the migration file created in db/migrate, right? If so I think that would be helpful context to add above.